assets/app/views/_volumes.html

```diff
@@ -46,5 +46,23 @@
       <dt ng-if-start="volume.gitRepo.revision">Revision:</dt>
       <dd ng-if-end>{{volume.gitRepo.revision}}</dd>
     </div>
+    <div ng-if="volume.downwardAPI">
+      <dt>Type:</dt>
+      <dd>downward API</dd>
+      <div ng-repeat="item in volume.downwardAPI.items">        
+        <dt>Volume file:</dt>
+        <dd>{{item.path}}&#8201;&#8594;&#8201;pod field "{{item.fieldRef.fieldPath}}"</dd>
+      </div>
+    </div>
+    <div ng-if="volume.configMap">
+      <dt>Type:</dt>
+      <dd>config map</dd>
+      <dt>Referenced config map:</dt>
+      <dd>{{volume.configMap.name}}</dd>
+      <div ng-repeat="item in volume.configMap.items">        
+        <dt>Projected key to path:</dt>
```

Comment:
What about "Volume file" instead of "Projected key to path", like in downward api?

The CLI doesn't show items, only the type and the referenced config map (called simply "Name").